Source/WarpX.cpp

```diff
@@ -1781,6 +1787,11 @@ WarpX::AllocLevelMFs (int lev, const BoxArray& ba, const DistributionMapping& dm
     Efield_fp[lev][1] = std::make_unique<MultiFab>(amrex::convert(ba,Ey_nodal_flag),dm,ncomps,ngEB,tag("Efield_fp[y]"));
     Efield_fp[lev][2] = std::make_unique<MultiFab>(amrex::convert(ba,Ez_nodal_flag),dm,ncomps,ngEB,tag("Efield_fp[z]"));
 
+    if (use_filter)
+    {
+        ngJ += bilinear_filter.stencil_length_each_dir - amrex::IntVect(1);
+    }
```

Comment:
Increasing the guard cells allocated for the current MultiFabs seems to be necessary in order for the CI test `bilinear_filter` to pass (otherwise the filtered current does not match the analytical and theoretical prediction).